clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -171,7 +171,29 @@
                     80
                 ]
             }
-        }
+        },
+        "keybindings": [
+            {
+                "key": "tab",
```

Comment:
is it possible to do like 
"(editorLangId == cobol || editorLangId == COBOL ) && !inSnippetMode" ?

if yes, then we can remove 2 objects /duplications here

Reply:
Not working :-(